.github/workflows/maven-verify.yml

```diff
@@ -25,9 +25,29 @@ on:
 
 jobs:
   build:
-    name: Verify
-    uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v1
-    with:
-      ff-goal: 'install'
-      verify-goal: 'install -P run-its'
-      verify-fail-fast: false
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        java: [8, 11, 17]
+        jdk: [temurin, zulu]
+      fail-fast: false
+
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 120
+    steps:
+    - name: Verify
+      uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v1
```

Comment:
it is shared workflow ... can not be uses as step

Reply:
We can extend  this workflow with artifacts upload support

Reply:
@slawekjaranowski
This change does not exit in this branch any longer.